libs/coin-evm/src/adapters.ts

```diff
@@ -47,92 +53,194 @@ export const transactionToEthersTransaction = (tx: EvmTransaction): ethers.Trans
 /**
  * Adapter to convert an Etherscan-like operation into a Ledger Live Operation
  */
-export const etherscanOperationToOperation = (
+export const etherscanOperationToOperations = (
   accountId: string,
   tx: EtherscanOperation,
-): Operation => {
+): Operation[] => {
```

Comment:
We could add in the method comment that one transaction actually translates into multiple operations in the live because smart contract thingie

Reply:
Done 👌